modules/KalturaSupport/kalturaIframeClass.php

```diff
@@ -1328,7 +1328,7 @@ function getIFramePageOutput( ){
 
 <script type="text/javascript">
     var customCSS = <?php echo $customCss ?>;
-    if (['kWidget'] && !window['kWidget'].isMobileDevice() && customCSS){
+    if (['kWidget'] && !window['kWidget'].isMobileDevice() && customCSS || window['kWidget'].isIOS() || window['kWidget'].isAndroid()){
```

Comment:
@ErezBouskila need to include test if mobile skin flag is enabled, as mobile device by itself is not sufficient